CONTRIBUTING.adoc

```diff
@@ -193,6 +193,103 @@ Follow these steps to ensure that virtual box interfaces are unmanaged:
 
     $ sudo nmcli con reload
 
+=== Develop and test using a docker-in-docker cluster
+
+It's possible to run an OpenShift multinode cluster on a single host
+via docker-in-docker (dind).  Cluster creation is cheaper since each
+node is a container instead of a VM.  This was implemented primarily
+to support multinode network testing, but may prove useful for other
+use cases.
+
+To run a dind cluster in a VM, follow steps 1-3 of the Vagrant
+instructions and then execute the following:
+
+        $ export OPENSHIFT_DIND_DEV_CLUSTER=true
+        $ vagrant up
+        $ vagrant ssh
+        $ cd /vagrant
+        $ hack/dind-cluster.sh start
```

Comment:
...
    Error response from daemon: Conflict. The name "openshift-master" is already in use by container 564ed56cf240. You have to delete (or rename) that container to be able to reuse that name.
    json: cannot unmarshal array into Go value of type types.ContainerJSON
    Error response from daemon: Conflict. The name "openshift-node-1" is already in use by container f5d7f21ce0de. You have to delete (or rename) that container to be able to reuse that name.
    json: cannot unmarshal array into Go value of type types.ContainerJSON

does it automatically get started after the first "vagrant up"? Or are those errors spurious? Or...?